packages/jest-runtime/src/__tests__/test_root/RegularModule.js

```diff
@@ -25,17 +25,18 @@ function getModuleStateValue() {
   return moduleStateValue;
 }
 
-(() => {
+const lazyRequire = () => {
```

Comment:
eslint was telling me about extra parenthesis which aren't necessary. This file is also an edge case, though, so this is fine too as long as that code doesn't run.